### PR TITLE
PMM-4928 Enable pmm2 testing repo for ami/ovf images for their build

### PR DESCRIPTION
**Pull request by hors**
https://github.com/Percona-Lab/pmm-submodules/pull/523
https://pmm.cd.percona.com/job/pmm2-ovf-slava/22/console

### CODE REVIEW

File: ansible/roles/ami-ovf/vars/main.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+    pmm2_server_release_repo: release
```
Comment (AlekSi) [inline ansible/roles/ami-ovf/vars/main.yml]: Variable name should not contain its value :) A better name would be `pmm2_server_repo_final` or something like that.

File: ansible/roles/ami-ovf/tasks/main.yml [hunk 2]
```diff
@@ -1,4 +1,23 @@
 ---
+    - name: Packages                   | Remove PMM2 Testing Server repository
+      yum_repository:
+        name: pmm2-server
+        state: absent
+
+    - name: Packages                   | Clean up yum metadata
+      command: yum clean metadata
```
Comment (AlekSi) [inline ansible/roles/ami-ovf/tasks/main.yml]: Does it make sense to use `yum clean all` there for extra safety?

File: ansible/roles/ami-ovf/vars/main.yml (hunk 1 above)
Comment (hors) [reply ansible/roles/ami-ovf/vars/main.yml]: fixed

File: ansible/roles/ami-ovf/tasks/main.yml (hunk 2 above)
Comment (hors) [reply ansible/roles/ami-ovf/tasks/main.yml]: I think it is ok to use 'yum clean metadata'.

**Pull request merged**